litmus-portal/graphql-server/pkg/database/mongodb/operations/project.go

```diff
@@ -77,20 +77,20 @@ func AddMember(ctx context.Context, projectID string, member *dbSchema.Member) e
 	return nil
 }
 
-//RemoveMember ...
-func RemoveMember(ctx context.Context, projectID string, userName string) error {
+//RemoveInvitation ...
+func RemoveInvitation(ctx context.Context, projectID string, userName string) error {
 	query := bson.M{"_id": projectID}
 	update := bson.M{"$pull": bson.M{"members": bson.M{"username": userName}}}
 	_, err := projectCollection.UpdateOne(ctx, query, update)
 	if err != nil {
-		log.Print("Error Removing the member with username:", userName, "from project with project id: ", projectID, err)
+		log.Print(err)
 		return err
 	}
 	return nil
 }
 
 //UpdateInvite ...
-func UpdateInvite(ctx context.Context, projectID, userName string, invitation dbSchema.Invitation) error {
+func UpdateInvite(ctx context.Context, projectID, userName string, invitation dbSchema.Invitation, Role *model.MemberRole) error {
```

Comment:
We should have the `role` parameter as an optional one here as we don't want to send the role for `acceptInvitation`. It can cause unnecessary role updates